build/yamls/antrea-ipsec.yml

```diff
@@ -3740,8 +3740,7 @@ data:
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
     #defaultMTU: 0
 
-    # Whether or not to enable IPsec encryption of tunnel traffic. IPsec encryption is only supported
-    # for the GRE tunnel type.
```

Comment:
Could you remove "tunnelType = gre" in line 3736 then?

Reply:
sure, will do.